docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -121,9 +76,19 @@ Our :ref:`debugging tips <debugging_modules>` will help if you run into bugs as
 Verifying your module code locally
 ----------------------------------
 
+The simplest way is to use ``ansible`` adhoc command:
+
+.. code:: shell
+
+    ansilbe -m library/my_test.py -a '... arguments' remotehost
+
 If your module does not need to target a remote host, you can quickly and easily exercise your code locally like this:
 
--  Create an arguments file, a basic JSON config file that passes parameters to your module so that you can run it. Name the arguments file ``/tmp/args.json`` and add the following content:
+.. code:: shell
+
+    ansilbe -m library/my_test.py -a '... arguments' localhost
+
+-  If for any reason you want to avoid going through Ansible, another way is to create an arguments file, a basic JSON config file that passes parameters to your module so that you can run it. Name the arguments file ``/tmp/args.json`` and add the following content:
```

Comment:
Could list reasons:
- print() is available for debugging.
- pdb works as expected.